### ISSUE TITLE
error in proces sentinel: nrepl-server-sentinel when trying to cider-jack-in from cider-nrepl repository

### ISSUE THREAD

**Issue by mallt** [clojure-emacs/cider-nrepl#554]
## Expected behavior
Running `cider-jack-in` from within the `cider-nrepl` repository launches CIDER.

## Actual behavior
Running `cider-jack-in` causes an error:
error in process sentinel: nrepl-server-sentinel: Could not start nREPL server: Warning: Clojure is excluded, assuming an appropriate fork (Clojure 1.7 or later) is provided.

## Steps to reproduce the problem
Run `cider-jack-in` from within the `cider-nrepl` repository 

### cider-nrepl version
latest master branch (commit 6a64a4392c8f6692172f63dbe9821ed95ee6ec8d)

### Java version
openjdk version "1.8.0_181"

### Operating system
ubuntu 16.04

**Comment by bbatsov** [comment]
Works fine for me. You chose lein, right?

**Comment by mallt** [comment]
Yes I choose lein. These are the errors that are printed to the `*Messages*` buffer:
```
Error loading clojure.tools.nrepl.server: Could not locate clojure/tools/nrepl/server__init.class or clojure/tools/nrepl/server.clj on classpath.
Error loading complete.core: Could not locate complete/core__init.class or complete/core.clj on classpath.
Exception in thread "main" java.lang.ClassNotFoundException: clojure.tools.nrepl.server, compiling:(/tmp/form-init2948801619982203135.clj:1:14083)
```

Should I include `clojure.tools.nrepl.server` and `complete.core` somewhere in my profiles perhaps?

Thanks!

**Comment by bbatsov** [comment]
No, that should not be necessary. What's your lein version? Do you have anything in your profiles?

**Comment by mallt** [comment]
My lein version is 2.7.0.

For CIDER I normally use a cider profile:
`(setq cider-lein-parameters "with-profile cider repl :headless")`

In the `cider-nrepl` repo I've set the cider profile to:
```clojure
{:cider [:dev
         {:dependencies [[org.clojure/clojure "1.7.0"]]}]}
```

I tried to add the `tools.nrepl` to this profile, and then `cider-jack-in` does not result in an error anymore:
```clojure
{:cider [:dev
         {:dependencies [[org.clojure/clojure "1.7.0"]
                         [org.clojure/tools.nrepl "0.2.13"]]}]}
```

So it seems the `tools.nrepl` dependency is needed somehow?
